README.md

```diff
@@ -63,6 +63,10 @@ every '0 0 27-31 * *' do
   command "echo 'you can use raw cron syntax too'"
 end
 
+every 1.minute do
```

Comment:
Will you also document `1.week`, `1.month`, `1.year`? Would it not be better to simply add a comment to the first example, similarly to the third, explaining that any unit of time supported by cron can be given?